third_party/terraform/resources/resource_compute_network_peering.go

```diff
@@ -57,15 +59,13 @@ func resourceComputeNetworkPeering() *schema.Resource {
 
 			"export_custom_routes": {
 				Type:        schema.TypeBool,
-				ForceNew:    true,
 				Optional:    true,
 				Default:     false,
 				Description: `Whether to export the custom routes to the peer network. Defaults to false.`,
 			},
 
 			"import_custom_routes": {
 				Type:        schema.TypeBool,
-				ForceNew:    true,
```

Comment:
Can these fields actually be updated? I don't see them being read at all during the update

Reply:
Should be! Here are log blurbs from running the test locally:

```
{
 "id": "7942091317901554066",
 "creationTimestamp": "2020-10-22T11:29:17.044-07:00",
 "name": "network-test-2-qt8r08lyeq",
 "selfLink": "https://www.googleapis.com/compute/v1/projects/<project>/global/networks/network-test-2-qt8r08lyeq",
 "autoCreateSubnetworks": false,
 "peerings": [
  {
   "name": "peering-test-2-qt8r08lyeq",
   "network": "https://www.googleapis.com/compute/v1/projects/<project>/global/networks/network-test-1-7187348105854718901",
   "state": "ACTIVE",
   "stateDetails": "[2020-10-22T11:29:42.074-07:00]: Connected.",
   "autoCreateRoutes": true,
   "exportCustomRoutes": false,
   "importCustomRoutes": false,
   "exchangeSubnetRoutes": true,
   "exportSubnetRoutesWithPublicIp": true,
   "importSubnetRoutesWithPublicIp": false
  }
 ],
 "routingConfig": {
  "routingMode": "REGIONAL"
 },
 "kind": "compute#network"
}

...

---[ REQUEST ]---------------------------------------
PATCH /compute/v1/projects/<project>/global/networks/network-test-2-qt8r08lyeq/updatePeering?alt=json&prettyPrint=false HTTP/1.1

{
 "networkPeering": {
  "exchangeSubnetRoutes": true,
  "exportCustomRoutes": true,
  "exportSubnetRoutesWithPublicIp": true,
  "importCustomRoutes": true,
  "name": "peering-test-2-qt8r08lyeq",
  "network": "https://www.googleapis.com/compute/v1/projects/<project>/global/networks/network-test-1-7187348105854718901"
 }
}

... 

GET /compute/v1/projects/<project>/global/networks/network-test-2-qt8r08lyeq?alt=json&prettyPrint=false HTTP/1.1


{
 "id": "7942091317901554066",
 "creationTimestamp": "2020-10-22T11:29:17.044-07:00",
 "name": "network-test-2-qt8r08lyeq",
 "selfLink": "https://www.googleapis.com/compute/v1/projects/<project>/global/networks/network-test-2-qt8r08lyeq",
 "autoCreateSubnetworks": false,
 "peerings": [
  {
   "name": "peering-test-2-qt8r08lyeq",
   "network": "https://www.googleapis.com/compute/v1/projects/<project>/global/networks/network-test-1-7187348105854718901",
   "state": "ACTIVE",
   "stateDetails": "[2020-10-22T11:29:55.363-07:00]: Connected.",
   "autoCreateRoutes": true,
   "exportCustomRoutes": true,
   "importCustomRoutes": true,
   "exchangeSubnetRoutes": true,
   "exportSubnetRoutesWithPublicIp": true,
   "importSubnetRoutesWithPublicIp": false
  }
 ],
 "routingConfig": {
  "routingMode": "REGIONAL"
 },
 "kind": "compute#network"
}

```